/server/Dockerfile doesn't run successfully

Typescript compiler needs to compile into a `build` folder before the server can be run but its currently impossible to for it to make the `build` directory because the root privileges was dropped for a non-root user in the [server Dockerfile](https://github.com/danBamikiya/cuttlink/blob/28584d1f2d978811cc449f18602ce54cf6ec6f43/server/Dockerfile) so there's a `error TS5033: Could not write file '/app/build/server.js': EACCES: permission denied, mkdir '/app/build'.`  crashing the run.

--- a/server/Dockerfile
+++ b/server/Dockerfile
@@ -3,21 +3,14 @@ FROM node:14-slim AS server-dev
 
 WORKDIR /app
 
-# create a non-root user 'cuttlink'
-RUN useradd -r -u 888 -g root -m cuttlink -c "Non-root user"
-
-# make non-root user 'cuttlink' the owner of WORKDIR
-RUN chown -R cuttlink:root /app
-
-COPY --chown=cuttlink:root ./package*.json ./
+COPY ./package*.json ./
 
 RUN npm i
 
-COPY --chown=cuttlink:root . .
+COPY . .
 
 EXPOSE 3000
 
-USER cuttlink
 
 ENV NODE_ENV development
 

